umh.docs.umh.app/content/en/docs/production-guide/security/firewall-rules.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Firewall Rules"
+menuTitle: "Firewall Rules"
+description: |
+  This page describes how to setup firewall rules for the UMH instances.
+content_type: task
+weight: 50
+---
+
+<!-- overview -->
+
+For closed enterprise networks with specific permissions, this page outlines firewall rule configuration
```

Comment:
Some enterprise networks might work in a whitelist manner, where by default all outgoing and incoming communication is blocked. Installing and maintaining the UMH requires however internet access, e.g., to download the Operating System, Docker containers, allow monitoring via the Management Console, or load 3rd party plugins. All of the dependencies are normally hosted on different servers, and might change at any time based on the vendors mood.

To simplify the user experience, we moved all mandatory services to a single domain. However, if you want to install 3rd party components (such as additional plugins for Node-RED or Grafana), you need to whitelist additional domains.